FW 5.12.1

danan72 opened the pull request:
Tłumaczenie FW 5.12.1

danan72 committed:
FW5.12.1

BeckyDTP commented on other/usr/share/locale/en_GB/LC_MESSAGES/blanket.translation:
```diff
@@ -33,8 +33,8 @@ No rect entry	Brak wpisu rect
-If you want to read or shop on your Kindle while continuing to charge over 	Jeśli chcesz czytać lub robić zakupy w trakcie ładowania przez USB, proszę
-USB, please keep the USB cable attached, but eject your Kindle from your 	 pozostawić podłączony przewód USB, ale bezpiecznie odłączyć Kindle
+If you want to read or shop on your Kindle while continuing to charge over 	Jeśli chcesz czytać lub robić zakupy w trakcie ładowania przez USB, to 
+USB, please keep the USB cable attached, but eject your Kindle from your 	pozostaw podłączony przewód USB, ale bezpiecznie odłącz swojego Kindle w 
```
Chyba usunąć „w” na końcu, bo już jest w następnym wersie.

danan72 replied:
Poprawione

BeckyDTP commented on other/usr/share/locale/en_GB/LC_MESSAGES/kb-gu.translation:
```diff
@@ -1 +1 @@
-Gujarati	Gudźarati
```
To była oficjalna nazwa: https://pl.wikipedia.org/wiki/J%C4%99zyk_gud%C5%BAarati

danan72 replied:
OK.

danan72 committed:
FW 5.12.1 drobne poprawki

danan72 committed:
FW 5.12.1 manual install KV & PW2

Paczki do ręcznej instalacji spolszczenia 5.12.1, przetestowane na Kindle Voyage i Paperwhite 2.
Paczki zawierają spolszczenie, polską klawiaturę i odblokowany tryb PDF reflow.

TODO -> Zwężenie marginesów, automatyczna instalacja przez MRPI, paczki dedykowane dla innych modeli (PW3, K8, itp.)

danan72 committed:
Revert "FW 5.12.1 manual install KV & PW2"

This reverts commit 5bb77dc0cd6c183a8c94e956a576ad68439a66da.

danan72 committed:
FW 5.12.1 poprawiona paczka do ręcznej instalacji na KV i PW2

Paczka zawiera przetestowane spolszczenie dla Kindle Voyage i Paperwhite 2 z polską klawiaturą i odblokowanym trybem PDF reflow.

Do zrobienia: zwężone marginesy, paczki do automatycznej instalacji, wersje dla innych modeli

danan72 commented:
> Tu jest drobny problem, bo w paczce dla PW2 jest spakowana wersja dla KV :)

Dziękuję za czujność. Już poprawione.

danan72 committed:
Wąskie marginesy dla KV i zmniejszone etykiety plików

danan72 committed:
FW 5.12.1 dla modelu Paperwhite 3

Pliki przygotowane na podstawie wersji dla Voyage, nieprzetestowane.
